lib/cunit/cu_pc_patch_lazperf.c

```diff
@@ -281,6 +289,47 @@ test_patch_filter_lazperf_zero_point()
 	pc_patch_free((PCPATCH*) pa);
 	pc_pointlist_free(pl);
 }
+
+static void
+test_patch_compression_with_multiple_dimension()
+{
+	PCPOINT *pt;
+	int i;
+	int npts = 5;
+	PCPOINTLIST *pl;
+	PCPATCH_LAZPERF *pal;
+	PCPATCH_UNCOMPRESSED *pau;
+	char *str1, *str2;
+
+	// build a list of points
+	pl = pc_pointlist_make(npts);
+
+	for ( i = 0; i < npts; i++ )
+	{
+		pt = pc_point_make(multipledimschema);
+		pc_point_set_double_by_name(pt, "x", i*2);
+		pc_point_set_double_by_name(pt, "y", i*1.9);
+		pc_point_set_double_by_name(pt, "Z", i*0.34);
```

Comment:
Minor, but is there any reason for using a capital Z and lower-case for x and y?